cli/cmd/cluster/cluster.go

```diff
@@ -30,12 +30,6 @@ import (
 	"github.com/kinvolk/lokomotive/pkg/platform"
```

Comment:
> All platform packages are now imported by utils.go file, so anonymous imports are no longer needed.

I am not sure if this commit message is valid. The anonymous imports are not needed anymore because we don't rely on the init function call registration method. So importing in `utils.go` or not does not matter.

Reply:
Good point. Fixed.